sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/RampupThrottlingFnTest.java

```diff
@@ -50,7 +49,7 @@ public class RampupThrottlingFnTest {
   @Mock private Counter mockCounter;
   private final Sleeper mockSleeper =
       millis -> {
-        verify(mockCounter).inc(millis);
```

Comment:
would you mind explain a little bit how this fixed the flaky test

Reply:
`verify` expected exactly one call, but sometimes the code retries and calls `inc(millis)` multiple times causing the test to fail.